ecs-cli/modules/cli/local/converter/converter.go

```diff
@@ -283,6 +283,7 @@ func convertLogging(logConfig *ecs.LogConfiguration) *composeV3.LoggingConfig {
 	if logConfig == nil {
 		return nil
 	}
+	log.Warningf("%s will not be supported", ecs.LogDriverAwslogs)
```

Comment:
should we only log this if the logConfig driver is set to awslogs ?

Reply:
done!